annlite/executor.py

```diff
@@ -273,6 +277,30 @@ def search(
 
             flat_docs.match(self._index, **match_args)
 
+    @requests(on='/backup')
+    def backup(self, target: Optional[str] = None, parameters: Dict = {}, **kwargs):
```

Comment:
```suggestion
    def backup(self,  parameters: Dict = {}, **kwargs):
```
All the request params should be included in the dict `parameters`

Reply:
When backing up outside the flow, also use `parameters` ?
```python
index = AnnLiteIndexer(...)
...
index.backup(parameters={'target': target})
```

Reply:
exactly.